Codeinwp/neve
# fix: search icon color inside sidebar

### Summary
Fixes Search icon component color.


### Will affect visual aspect of the product

NO

### Test instructions


- Add the Search Icon to the Mobile Sidebar.
- Change the Color and Hover color.
- Check the changes both in front-end and customizer.


Closes #1985.
